server/app/src/androidTest/java/sh/calaba/instrumentationbackend/StatusReporterActivity.java

```diff
@@ -68,15 +64,10 @@ public void onCreate(Bundle savedInstanceState) {
     }
 
     private void reportFailure(String message) throws IOException {
-        clearFailure();
-
-        OutputStream fileOutputStream = openFileOutput(FAILURE_FILE_PATH, Context.MODE_WORLD_READABLE);
+        System.out.println("Failure state: " + message);
 
-        try {
-            fileOutputStream.write(message.getBytes());
-        } finally {
-            fileOutputStream.close();
-        }
+        clearFailure();
```

Comment:
Just a note - in theory clearFailure() should not be needed. dumpPublicFile() overwrites what ever is in there to begin with.

Reply:
Yeah... This code was here before, but it seems unnecessary now

Reply:
I will add explicit second parameter = false for FileOutputStream just to make sure that append is disabled 😃